h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -1087,6 +1087,8 @@ \subsubsection{Proximal Operators}
 given solution. There are many potential uses: for example, it can be used as part of an ADMM consensus algorithm
 to obtain a unified solution over separate H2O clouds or in Bayesian regression approximation.
 
+\subsubsection{Cold Start}
+\texttt{cold\_start} allows you to specify whether the GLM model should be built from scratch. This parameter is only applicable when building a GLM model with multiple alpha/lambda values. If false and for a fixed alpha value, the next model with the next lambda value out of the lambda array will be built using the coefficients and the GLM state values of the current model. If true, the next GLM model will be built from scratch. The default value is false.
 
```

Comment:
@hannah-tillman Totally my fault.  Please add the following to some where

If an alpha array is specified and for a branch new alpha, the model will be built from scratch regardless of the value of cold_start.